CONTRIBUTING.md

```diff
@@ -2,8 +2,8 @@
 
 ### Setup
 ```sh
-$ git clone https://github.com/babel/babili
-$ cd babili
+$ git clone https://github.com/babel/minify
+$ cd minify
 $ npm install
 $ npm run bootstrap
```

Comment:
aside: this should be `npm run bootstrap` which also runs `yarn` beforehand? don't need npm install since we are using yarn